lib/isDocker.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+// Source Code from: https://github.com/sindresorhus/is-docker/blob/main/index.js
```

Comment:
why don't you add it as a dependency?

Reply:
Because `is-docker` library is `esm`.
As a dependency, an error will be reported

Reply:
install v2.2.1
it was commonjs and has the same code.